lib/widgets/content.dart

```diff
@@ -967,9 +970,22 @@ class AvatarImage extends StatelessWidget {
       null          => null, // TODO(#255): handle computing gravatars
       var avatarUrl => store.tryResolveUrl(avatarUrl),
     };
-    return (resolvedUrl == null)
-      ? const SizedBox.shrink()
-      : RealmContentNetworkImage(resolvedUrl, filterQuality: FilterQuality.medium, fit: BoxFit.cover);
+
+    if(resolvedUrl == null) {
+      return const SizedBox.shrink();
+    } else{
+      final avatarUrl = AvatarUrl.fromUserData(
```

Comment:
Instead of `else`, just let the normal non-null case come after the `if` statement entirely:

```suggestion
      return const SizedBox.shrink();
    }

    final avatarUrl = AvatarUrl.fromUserData(
```

The null case already returns, so this has the same effect.

This is called the "early return" pattern. It's helpful whenever you have a trivial case, or an error case, and then a normal or non-trivial case that's where the bulk of the interesting logic is. This way, the normal case with the interesting logic avoids getting indented deeper, and instead reads nicely straight down in the same column throughout the function. That's helpful for keeping the reader's attention on the main narrative of the function when that's what they want to be looking at, and for helping the reader systematically look at each of the error cases in turn when that's what they want to be looking at.